todo1/src/components/TodoList.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { addTodo, toggleTodo} from '../actions';
+import { connect } from 'react-redux';
+
+class TodoListContainer extends React.Component {
+    constructor() {
+      super();
+      this.state = {
```

Comment:
instead you can write 
```
state = {
 newTodo: ''
}
```
it is exactly the same thing, take a look at babel transpiles

[The constructor is dead, long live the constructor!](https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599)